README.md

```diff
@@ -54,7 +54,7 @@ Options can be any of the following command-line switches:
 Each animation file can be preceded by any combination of the following command-line switches:
```

Comment:
Indent the command-line switch documentation using four spaces, so that it appears as a code block with a fixed-width font.

Reply:
You should comment the line which is actually affected. GitHub will notice then when your review is outdated.